### ISSUE TITLE
globus_gsi_cert_utils_error.c:42: possible missing "," ?

### ISSUE THREAD

**Issue by dcb314** [gridcf/gct#182]
I just tried to compile the package with new C compiler clang-14.
It said:

globus_gsi_cert_utils_error.c:42:11: warning: suspicious concatenation of string literals in an array initialization; did you mean to separate the elements with a comma? [-Wstring-concatenation]

Source code is

/* 7 */   "Proxy does not comply with proxy certificate standard"
/* 8 */   "Error determining certificate type"

**Comment by fscheiner** [comment]
Indeed, there is a comma missing between the last two elements:

https://github.com/gridcf/gct/blob/5882044a0eac96b38b276f0ed4f565afc16fe494/gsi/cert_utils/source/library/globus_gsi_cert_utils_error.c#L30-L43

Thanks for the pointer, @dcb314!

**Comment by fscheiner** [comment]
@dcb314:
Apart from that, does compilation work well with clang-14? So far we only ever used gcc for compilation IIC. And it's good to know if clang would work, too.

BTW, are you compiling on GNU/Linux or *BSD?

**Comment by dcb314** [comment]
>Apart from that, does compilation work well with clang-14?

Yes. The warning I mentioned is the only one in the compilation of the whole package.

>BTW, are you compiling on GNU/Linux or *BSD?

Fedora 36 beta, so GNU/Linux.

**Comment by fscheiner** [comment]
Nice! Thanks for the valuable information.
